src/main/java/com/example/bank/service/demo/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package com.example.bank.service.demo.service.impl;
+
+import com.example.bank.service.demo.entity.User;
+import com.example.bank.service.demo.repository.UserRepository;
+import com.example.bank.service.demo.service.UserService;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class UserServiceImpl implements UserService {
+    private UserRepository userRepository;
+
+    @Override
+    public User save(User user) {
+        return userRepository.save(user);
+    }
+
+    @Override
+    public User update(User user) {
+        return userRepository.save(user);
+    }
+
+    @Override
+    public void delete(User user) {
+        userRepository.delete(user);
+    }
+
+    @Override
+    public User getById(Long id) {
+        return userRepository.getOne(id);
```

Comment:
You can use findById(), it's default method which return Optional and in this method you can return User or throw exception

Reply:
Thanks for advice. but i find this realization ok.